packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -110,6 +111,9 @@ class CupertinoScrollbar extends RawScrollbar {
   /// dragging the scrollbar.
   final Radius radiusWhileDragging;
 
+  /// Dictates the side of the screen the scrollbar appears in.
+  final TextDirection? textDirection;
```

Comment:
> Hmm, the textDirection only has a `ltr` and `rtl`. Can you suggest how this top, bottom feature be achieved?🤔 Would it require a new set of `utd` , `dtu` properties? 🤔

I don't know what the solution is, but I do think TextDirection is a bit misleading. As a user, I do not understand why TextDirection affects the placement of the Scrollbar. What does this mean that it dictates the side of the screen that the Scrollbar appears on? What does it mean to use ltr vs. rtl? This in addition to the horizontal axis being unaffected.

Reply:
Maybe changing the property name to something more intuitive and elaborating the docs would help?🤔 Though the horizontal scroll thing would still persist...

Reply:
Yeah... Maybe you could create a cross axis orientation, and then pass that on to the ScrollbarPainter, where you could implement which side to paint on, regardless of the axis.

Reply:
Okay thanks :smile: . I'll try to implement it as soon as I'm free. :)

Reply:
Hey @Piinks, sorry for the delay, was a bit busy with exams.
I was thinking about implementing something like `AxisOrientation` in `basic_types.dart` and then pass on this property to ScrollPainter. There we may use this property to modify the x,y cords of the `_thumbRect`.
What are your views on this?

Reply:
No worries! Thank you for continuing to work on this. I think that makes sense. I would be happy to take a look. :)